api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+            'fr' => 'Afficher les informations de base sur les utilisateurs de n\'importe quelle équipe'
+        ],
+        'view-team-teamUsersBasicInfo' => [
+            'en' => 'View Teams Team Users Basic Info',
+            'fr' => 'Afficher les informations de base sur les utilisateurs de l\'équipe'
+        ],
+        'create-any-team' => [
+            'en' => 'Create Any Team',
+            'fr' => 'Créer n\'importe quelle équipe'
+        ],
+        'update-any-team' => [
+            'en' => 'Update Any Team',
+            'fr' => 'Mise à jour de n\'importe quelle équipe'
+        ],
+        'update-team-team' => [
+            'en' => 'Update Team Team',
+            'fr' => 'Mise à jour de l\'équipe'
+        ],
+        'delete-any-team' => [
+            'en' => 'Delete Any Team',
+            'fr' => 'Supprimer une équipe'
+        ],
+
+        'view-any-role' => [
+            'en' => 'View Any Role',
+            'fr' => 'Voir tous les rôles'
+        ],
+        'assign-any-role' => [
+            'en' => 'Assign Any Role',
+            'fr' => 'Attribuer n\'importe quel rôle'
+        ],
+        'assign-team-role' => [
+            'en' => 'Assign Team Role',
+            'fr' => 'Attribuer un rôle à l\'équipe'
+        ],
+        'update-any-role' => [
+            'en' => 'Update Any Role',
+            'fr' => 'Mise à jour de n\'importe quel rôle'
+        ],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Roles
+    |--------------------------------------------------------------------------
+    |
+    | A map of all roles with the shape:
+    |
+    |   $name => [
+    |       'display_name' => LocalizedString,
+    |       'description' => LocalizedString,
+    |   ]
+    |
+    */
+    'roles' => [
+        'guest' => [
+            'display_name' => [
+                'en' => 'Guest',
+                'fr' => 'Guest'
+            ],
+            'description' => [
+                'en' => 'These permissions are available to anyone, even not logged in.',
+                'fr' => 'Ces autorisations sont accessibles à tous, même aux personnes non connectées.'
+            ]
+        ],
+
+        'base_user' => [
+            'display_name' => [
+                'en' => 'Base User',
+                'fr' => 'Utilisateur de base'
+            ],
+            'description' => [
+                'en' => 'Available to any logged-in user.',
+                'fr' => 'Disponible pour tout utilisateur connecté.'
+            ]
+        ],
+
+        'applicant' => [
+            'display_name' => [
+                'en' => 'Applicant',
+                'fr' => 'Candidat'
+            ],
+            'description' => [
+                'en' => 'Can edit their own profile and apply to jobs.',
+                'fr' => 'Ils peuvent modifier leur propre profil et postuler à des emplois.'
+            ]
+        ],
+
+        'team_admin' => [
```

Comment:
Could this be just `admin` or are there plans to add other types of admin roles (other than `super_admin`)?